src/components/button/button.js

```diff
@@ -0,0 +1,70 @@
+// javascript 'shim' to trigger the click event of element(s)
```

Comment:
Being a little bit fussy here, but can take this opportunity to wrap comments consistently and use the proper capitalisation?